packages/components/src/translate.test.js

```diff
@@ -11,7 +11,6 @@ describe('getCurrentLanguage', () => {
 		i18next.language = originalLang;
 	});
 	it('should return the locale', () => {
-		expect(i18next.language).toBe('en');
```

Comment:
The i18n mock doesn't have a language at start. And the purpose is to test the default return value, then a i18next value.